fhir-persistence-cos/src/main/java/com/ibm/fhir/persistence/cos/client/COSPropertiesAdapter.java

```diff
@@ -72,4 +72,9 @@ public int getMaxKeys() {
             return defaultMaxKeys();
         }
     }
+
+    @Override
+    public boolean isCompress() {
+        return "Y".equalsIgnoreCase(properties.getProperty(COSPropertyConstants.COS_COMPRESS));
```

Comment:
don't we typically use true/false in our property values?

Reply:
I used the same pattern I saw for cos.credential.ibm when used in property files. I can make it more flexible so we support both.

Reply:
Also changed for cos.credential.ibm in the same adapter.